CMakeLists.txt

```diff
@@ -23,13 +23,17 @@ target_link_libraries(boost_unordered
     Boost::throw_exception
 )
 
-if(CMAKE_VERSION VERSION_GREATER 3.18 AND CMAKE_GENERATOR MATCHES "Visual Studio")
-
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.12)
```

Comment:
You don't need this check.

Reply:
I mean, CONFIGURE_DEPENDS "works" on pre-3.12, because it's interpreted as a path, and doesn't do anything.